schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -1592,6 +1596,212 @@
         "settings",
         "protectedSettings"
       ]
+    },
+    "VMSnapshot":{
```

Comment:
Could you link the line of definition? I see location defined for 3 resources, but extensions is not one of them. I ask because "tags" is a property of the snapshot extension, but I can't find it defined for "extensions".